From 0f8261015380be71f8a63c8aa37926c2a32d1870 Mon Sep 17 00:00:00 2001
From: Umut Tezduyar Lindskog <umuttl@axis.com>
Date: Wed, 8 Jul 2015 14:35:32 +0200
Subject: [PATCH] property callback returns are consistent

It is no different to return 0 over 1 in the property
callback. It is confusing to return 1 which made me think
1 has a special purpose. This way code is consistent with
the rest of the tree.
---
 src/login/logind-user-dbus.c       |  6 +-----
 src/machine/machine-dbus.c         | 12 ++----------
 src/network/networkd-network-bus.c |  6 +-----
 3 files changed, 4 insertions(+), 20 deletions(-)

diff --git a/src/login/logind-user-dbus.c b/src/login/logind-user-dbus.c
index 0f72d70..36c0e86 100644
--- a/src/login/logind-user-dbus.c
+++ b/src/login/logind-user-dbus.c
@@ -103,11 +103,7 @@ static int property_get_sessions(
 
         }
 
-        r = sd_bus_message_close_container(reply);
-        if (r < 0)
-                return r;
-
-        return 1;
+        return sd_bus_message_close_container(reply);
 }
 
 static int property_get_idle_hint(
diff --git a/src/machine/machine-dbus.c b/src/machine/machine-dbus.c
index 7813a0b..38a4c7b 100644
--- a/src/machine/machine-dbus.c
+++ b/src/machine/machine-dbus.c
@@ -61,11 +61,7 @@ static int property_get_id(
         assert(reply);
         assert(m);
 
-        r = sd_bus_message_append_array(reply, 'y', &m->id, 16);
-        if (r < 0)
-                return r;
-
-        return 1;
+        return sd_bus_message_append_array(reply, 'y', &m->id, 16);
 }
 
 static int property_get_state(
@@ -112,11 +108,7 @@ static int property_get_netif(
 
         assert_cc(sizeof(int) == sizeof(int32_t));
 
-        r = sd_bus_message_append_array(reply, 'i', m->netif, m->n_netif * sizeof(int));
-        if (r < 0)
-                return r;
-
-        return 1;
+        return sd_bus_message_append_array(reply, 'i', m->netif, m->n_netif * sizeof(int));
 }
 
 static BUS_DEFINE_PROPERTY_GET_ENUM(property_get_class, machine_class, MachineClass);
diff --git a/src/network/networkd-network-bus.c b/src/network/networkd-network-bus.c
index b5f8f5c..5717a15 100644
--- a/src/network/networkd-network-bus.c
+++ b/src/network/networkd-network-bus.c
@@ -53,11 +53,7 @@ static int property_get_ether_addrs(
                         return r;
         }
 
-        r = sd_bus_message_close_container(reply);
-        if (r < 0)
-                return r;
-
-        return 1;
+        return sd_bus_message_close_container(reply);
 }
 
 const sd_bus_vtable network_vtable[] = {
-- 
2.4.5

